integration-tests/library/src/main/java/com/squareup/anvil/test/RepeateableContributions.kt

```diff
@@ -0,0 +1,21 @@
+package com.squareup.anvil.test
```

Comment:
Filename has a mis-spelling. Repeatable vs Repeateable.

Reply:
I had to read this a couple times to see the typo 😁 .